changelogs/unreleased/gh-10477-fix-iproto-override-memleak.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/box
+
+- Fixed a memory leak in iproto override introduced in 3.1 (gh-10477).
```

Comment:
- Please remove "introduced in 3.1".
- Unclear when the leak happens. Better write: "Fixed a memory leak in IPROTO on execution of the callback installed with `box.iproto.override()`".

Reply:
Fixed.